packages/shared/src/hooks/squads/useSquadActions.ts

```diff
@@ -55,6 +62,23 @@ export const useSquadActions = ({
     onSuccess: () => client.invalidateQueries(membersQueryKey),
   });
 
+  const { mutateAsync: collapseSquadPinnedPosts } = useMutation(
+    collapsePinnedPosts,
+    {
+      onSuccess: () => {
+        client.invalidateQueries(feedQueryKey);
```

Comment:
Generally it's best practice to perform optimistic updates if we know what the mutation will be.
In this case we could manually update the cache instead of simply invalidating it.

Reply:
I agree with @DragosIorgulescu in this case invalidation is better then optimistic update since it is not something crucial to the user.  As long as we validate actual pin is applied (no api caching is involved).